winpr/libwinpr/nt/test/TestNtCreateFile.c

```diff
@@ -60,23 +54,23 @@ int TestNtCreateFile(int argc, char* argv[])
 	}
 
 	result = 0;
-
 out:
-
+	_RtlFreeUnicodeString(&uString);
```

Comment:
uString is never initialized if _RtlAnsiStringToUnicodeString fails and we will end up freeing an invalid pointer here

Reply:
Wrong, see line 25 (_RtlAnsiStringToUnicodeString)